stable/redis-ha/values.yaml

```diff
@@ -51,6 +51,11 @@ haproxy:
   init:
     resources: {}
 
+ # Enable if you want a dedicated port in haproxy for redis-slaves
+hapreadport:
```

Comment:
I don't think this necessarily needs it's own section and makes more sense if it falls under the `haproxy` yaml block. Something like:
```
haproxy:
  enabled: false
  readOnlyPort:
    enabled: true
    port: 6380
```

Reply:
sure, will do the changes and commit again. Thanks

Reply:
@ssalaues updated the template with required changes